ci-operator/step-registry/ipi/conf/vsphere/customize-resource/ipi-conf-vsphere-customize-resource-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: ipi-conf-vsphere-customize-resource
+  from: upi-installer
+  commands: ipi-conf-vsphere-customize-resource-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: CONTROL_PLANE_REPLICAS
+    default: "3"
+    documentation: |-
+      the number of the master nodes for ipi on vsphere cluster. cpms enabled in 4.15 with tp and 4.16+, the CONTROL_PLANE_REPLICAS can only set to 3 if version > 4.16 or 4.15 with tp.
+  - name: COMPUTE_NODE_REPLICAS
+    default: "3"
+    documentation: |-
+      the number of the worker nodes for ipi on vsphere cluster.
+  - name: CONTROL_PLANE_DISK_SIZE
```

Comment:
Suggest not to leave empty as the default value for these ENV vars.

Reply:
i tested with other job.if no default value set for these ENV. vm also have default value :4cpu, 16G memory, 120G disk

Reply:
VMs definitely have a default values for cpu/mem/disk even we do not set them explicitly in install-config.yaml. Here the step is for customized resource purpose, so it sounds more reasonable to set some  values (which is not equal to the default cpu/mem/disk values when no any set in install-config.yaml) for these ENV var. 

If we leave empty values as default for these env, the install-config.yaml will set cpu/mem/disk to empty in install-config.yaml, that looks strange.